include/PetriEngine/Colored/ColoredPetriNetBuilder.h

```diff
@@ -126,6 +129,7 @@ namespace PetriEngine {
         std::vector<Colored::Place> _places;
         std::vector<Colored::Transition> _transitions;
         std::vector<Colored::Arc> _inhibitorArcs;
+        std::vector<Colored::Variable> _variables;
```

Comment:
Jeg tror det er bedst vi bruger pointers her, så der ikke kommer til at findes flere af den samme variable i memory. Altså
```suggestion
        std::vector<const Colored::Variable *> _variables;
```